mecanum-drive/robot.py

```diff
@@ -6,52 +6,38 @@
 
 import wpilib
 from wpilib.drive import MecanumDrive
+from wpilib import Joystick, PWMSparkMax, TimedRobot
 
 
-class MyRobot(wpilib.TimedRobot):
+class MyRobot(TimedRobot):
     # Channels on the roboRIO that the motor controllers are plugged in to
-    frontLeftChannel = 2
-    rearLeftChannel = 3
-    frontRightChannel = 1
-    rearRightChannel = 0
+    kFrontLeftChannel = 2
+    kRearLeftChannel = 3
+    kFrontRightChannel = 1
+    kRearRightChannel = 0
 
     # The channel on the driver station that the joystick is connected to
-    joystickChannel = 0
+    kJoystickChannel = 0
 
     def robotInit(self):
-        """Robot initialization function"""
-        self.frontLeftMotor = wpilib.Talon(self.frontLeftChannel)
-        self.rearLeftMotor = wpilib.Talon(self.rearLeftChannel)
-        self.frontRightMotor = wpilib.Talon(self.frontRightChannel)
-        self.rearRightMotor = wpilib.Talon(self.rearRightChannel)
-
-        # invert the left side motors
-        self.frontLeftMotor.setInverted(True)
+        self.frontLeft = PWMSparkMax(self.kFrontLeftChannel)
+        self.rearLeft = PWMSparkMax(self.kRearLeftChannel)
+        self.frontRight = PWMSparkMax(self.kFrontRightChannel)
+        self.rearRight = PWMSparkMax(self.kRearRightChannel)
 
+        # invert the right side motors
         # you may need to change or remove this to match your robot
-        self.rearLeftMotor.setInverted(True)
-
-        self.drive = MecanumDrive(
-            self.frontLeftMotor,
-            self.rearLeftMotor,
-            self.frontRightMotor,
-            self.rearRightMotor,
-        )
-
-        self.drive.setExpiration(0.1)
+        self.frontRight.setInverted(True)
+        self.rearRight.setInverted(True)
 
-        self.stick = wpilib.Joystick(self.joystickChannel)
+        self.m_drive = MecanumDrive(self.frontLeft, self.rearLeft, self.frontRight, self.rearRight)
 
-    def teleopInit(self):
-        self.drive.setSafetyEnabled(True)
+        self.m_stick = Joystick(self.kJoystickChannel)
```

Comment:
Our examples drop the `m_` prefix as it's already clear that these are member variables by the use of `self`. We settled on this in the frc-docs PR too.

Reply:
on it.